project/plugins.sbt

```diff
@@ -0,0 +1,18 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.13.1")
+
+addSbtPlugin("org.musigma" % "sbt-rat" % "0.7.0")
```

Comment:
Only files in src/main/g8 are end up in the resulting stuff. This file defines plugins for just this repo. Since there isn't a plugins file in src/main/g8, there will be no plugins in the generated sbt project.